analyses/0013-litmuschaos/litmuschaos-analysis.md

```diff
@@ -0,0 +1,720 @@
+
+This analysis measures documentation against CNCF project maturity standards,
+and suggests possible improvements. In most cases there is more than one way to
+do things. Few recommendations here are meant to be prescriptive. Rather, the
+recommended implementations represent the reviewers' experience with how to
+apply documentation best practices. In other words, borrowing terminology from
+the lexicon of [RFCs][rfc-spec], the changes described here should be understood
+as "recommended" or "should" at the strongest, and "optional" or "may" in many
+cases. Any "must" or "required" actions are clearly denoted as such, and pertain
+to legal requirements such as copyright and licensing issues.
+
+## Project documentation
+
+LitmusChaos is a **graduated** project of CNCF. This means that the project
+should have [_very high_][criteria] standards for documentation.
+
+| Criterion                  |                      |
+| -------------------------- | -------------------- |
+| Information architecture   | 2. Needs improvement |
+| New user content           | 2. Needs improvement |
+| Content maintainability    | 2. Needs improvement |
+| Content creation processes | 2. Needs improvement |
+| Inclusive language         | 2. Needs improvement |
+
+### Comments
+
+The following sections contain brief assessments of each element of the Project
+Documentation rubric.
+
+Organization of the [doc page](https://docs.litmuschaos.io/) at the top level
+is unconventional. The Documentation tab in the banner menu has several
+graphically displayed options. The main one leads to a documentation main page
+that has tiled options organized in groups:
+
+- Explore using Litmus
+- Litmus for Advanced Users
+- More Resources
+
+There is also a Versions drop-down on the doc main page. Selecting the most
+recent version (3.91) brings up the "What is Litmus" page (the first page in the
+first ToC heading).
+
+#### Information architecture
+
+There is high level, conceptual “About” content. This content is split between
+at least two sections, "Introduction" and "Concepts".
+
+The main documentation site seems feature-complete. However, the documentation
+resides on several websites. The tutorials have their own site
+
+The documentation for the [API][api-site] is on its own website and is 
+maintained from the main project repository. The API documentation is 
+incomplete, containing very sparse semantic information about the API endpoints,
+objects, and actions.
+
+There are step-by-step instructions for most important functionality, including
+installation, configuration, and running a "first-time" experiment.
+
+- Formatting and organization of the instructions is inconsistent.
+- Some minor functionality does not have complete step-by-step instructions.
+  For example, a link in the instructions to connect an external delegate in
+  [Schedule a chaos scenario](https://v2-docs.litmuschaos.io/docs/user-guides/schedule-workflow)
```

Comment:
This seems to be an older website reference from the period where LitmusChaos v2 had recently been released and v1 was also very much in use. Hence, the content here isn't updated. This page has been updated since and we'll remove this along with any other unused sites.